src/main.rs

```diff
@@ -52,9 +52,9 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let client = Client::builder().build::<_, hyper::Body>(https);
 
     let mut handles = vec![];
-    let address = config.addresses.clone();
+    //let address = config.addresses.clone();
 
-    for u in address {
+    for u in &config.addresses {
```

Comment:
error[E0597]: `config.addresses` does not live long enough
  --> src/main.rs:57:14
   |
57 |     for u in &config.addresses {
   |              ^^^^^^^^^^^^^^^^^
   |              |
   |              borrowed value does not live long enough
   |              argument requires that `config.addresses` is borrowed for `'static`
...
74 | }
   | - `config.addresses` dropped here while still borrowed

For more information about this error, try `rustc --explain E0597`.